src/js/components/SearchPage.js

```diff
@@ -75,20 +88,55 @@ export default function SearchPage() {
     SEARCH_PAGE_SIZE
   )
 
+  const isResourceSearch = activeFacets["type"].includes(LR_TYPE_RESOURCEFILE)
+
   return (
     <div className="search-page w-100">
       <SearchBox value={text} onChange={updateText} onSubmit={onSubmit} />
       <div className="container">
         <div className="row">
-          <div className="col-3 mt-3 mt-lg-6">
-            <FilterableFacet
-              results={facetOptions("topics")}
-              name="topics"
-              title="Topics"
-              currentlySelected={activeFacets["topics"] || []}
-              onUpdate={onUpdateFacets}
-            />
+          {isResourceSearch ? null : <div className="col-3 mt-3 mt-lg-6"></div>}
+          <div className="search-results col-12 col-lg-8 col-xl-8 px-3 px-md-5 mt-3 mt-lg-6 mx-auto">
+            <div className="search-toggle">
+              <ul className="nav">
+                <li className="nav-item">
+                  <a
+                    className={`nav-link search-nav ${
+                      isResourceSearch ? "" : "active"
+                    }`}
+                    href="#"
+                    onClick={async () => toggleResourceSearch(false)}
+                  >
+                    Courses
+                  </a>
+                </li>
+                <li className="nav-item">
+                  <a
+                    className={`nav-link search-nav ${
+                      isResourceSearch ? "active" : ""
+                    }`}
+                    href="#"
+                    onClick={async () => toggleResourceSearch(true)}
+                  >
+                    Resources
+                  </a>
+                </li>
+              </ul>
+            </div>
           </div>
+        </div>
+        <div className="row">
+          {isResourceSearch ? null : (
+            <div className="col-3 mt-3 mt-lg-6">
+              <FilterableFacet
+                results={facetOptions("topics")}
+                name="topics"
+                title="Topics"
+                currentlySelected={activeFacets["topics"] || []}
+                onUpdate={onUpdateFacets}
+              />
+            </div>
```

Comment:
on page load, if I have `type=resourcefile` in the URL search params, I see a flash of the topics sidebar for a second. this is happening because `activeFacets` is stored in a `useState` hook in the `useCourseSearch` hook, but it is initialized with a default value which will be returned on the first render call. Then a `useEffect` callback fires and pulls the facets out of the URL bar and the value is updated, so here in our `SearchPage` component the facets will be empty for one render and then correct from then on:

![Screenshot from 2020-09-28 10-23-52](https://user-images.githubusercontent.com/6207644/94444879-c81b5280-0174-11eb-9de4-815ff78ce6fc.png)

(I just added a `console.log(activeFacets)` to `SearchPage.js`).

One way to get rid of the flash would be to change the ternary here to be

```js
isResourceSearch || !completedInitialLoad ? null : ...
```

we might want to do that here, and I can look into trying to fix that issue in the `useCourseSearch` hook so that it always returns a good value.